$NetBSD: patch-mosquitto.conf,v 1.8 2025/03/08 13:59:13 gdt Exp $

Align pid_file to pkgsrc norms.

Provide a dir for capath, but do not set it.  (In mosquitto, having a
key/cert for the server is linked to having a CA, and to treating
clients with a certificate from a known CA as authorized.  This is
complicated, and not about pkgsrc, and this change does not intend to
step into the situation.)

Log to syslog, instead of (perhaps) not logging.

--- mosquitto.conf.orig	2025-03-06 16:25:31.000000000 +0000
+++ mosquitto.conf
@@ -167,7 +167,7 @@
 # This should be set to /var/run/mosquitto/mosquitto.pid if mosquitto is
 # being run automatically on boot with an init script and
 # start-stop-daemon or similar.
-#pid_file
+pid_file @VARBASE@/run/mosquitto/mosquitto.pid
 
 # Set to true to queue messages with QoS 0 when a persistent client is
 # disconnected. These messages are included in the limit imposed by
@@ -361,7 +361,7 @@
 # "openssl rehash <path to capath>" each time you add/remove a certificate.
 # capath is not supported for websockets.
 #cafile
-#capath
+#capath @SSLCERTS@
 
 
 # If require_certificate is true, you may set use_identity_as_username to true
@@ -469,7 +469,7 @@
 # Note that if the broker is running as a Windows service it will default to
 # "log_dest none" and neither stdout nor stderr logging is available.
 # Use "log_dest none" if you wish to disable logging.
-#log_dest stderr
+log_dest syslog
 
 # Types of messages to log. Use multiple log_type lines for logging
 # multiple types of messages.
